### PR TITLE
Remove circular dependency

### PR DESCRIPTION
**Pull request by toddthomas**
resque-multi-step had an explicit dependency on itself for some unknown reason. I first tried to fix that simply by removing those dependency lines from the gemspec, but I had problems trying to build the gem using the existing jeweler infrastructure, so I decided to take this opportunity to switch to bundler's gem-building capabilities (`bundle gem`).

### CODE REVIEW

**Comment by dlovellrw** [comment]
:+1:

File: resque-multi-step.gemspec [hunk 1]
```diff
@@ -1,77 +1,26 @@
...
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'resque/plugins/multi_step_task/version'
 
-Gem::Specification.new do |s|
-  s.name = "resque-multi-step"
-  s.version = "2.0.8"
+Gem::Specification.new do |spec|
+  spec.name          = 'resque-multi-step'
+  spec.version       = Resque::Plugins::MultiStepTask::VERSION
+  spec.authors       = ['Peter Williams', 'Morgan Whitney', 'Jeff Gran', 'Cameron Mauch']
```
Comment (tracehelms) [inline resque-multi-step.gemspec]: `, 'Todd Thomas']` ???

**Comment by tracehelms** [comment]
:+1:

File: resque-multi-step.gemspec (hunk 1 above)
Comment (toddthomas) [inline resque-multi-step.gemspec]: Nah. I just changed the way the gem is built.

**Comment by jeffgran** [comment]
:+1:

**Comment by zastruga** [comment]
:+1:

**Pull request merged**